### PR TITLE
[DOCS]  feature transition section

### PR DESCRIPTION
**Pull request by kblaszczak-intel**

### CODE REVIEW

File: docs/Documentation/openvino_legacy_features.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+|   *Old solution:* POT discontinuation planned for 2024
+|    
+|   Neural Network Compression Framework (NNCF) now offers the same functionality as POT,
+    apart from its original feature set. It is currently the default tool for performing 
+    both, post-training and quantization optimizations, while POT is considered deprecated.
+ 
+|   :doc:`See how to use NNCF for model optimization <openvino_docs_model_optimization_guide>`
+|   `Check the NNCF GitHub project, including documentation <https://github.com/openvinotoolkit/nncf>`__
+
+| **new API 2.0**
```
Comment (tsavina) [inline docs/Documentation/openvino_legacy_features.md:86]: this header doesn't follow the name pattern, where the header is the old solution, kind of gives impression that we're removing new api

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_Tutorials.md [hunk 2]
```diff
@@ -31,22 +31,23 @@
...
+cases and may not reflect your particular needs.
 Before studying the tutorials, try to convert the model out-of-the-box by specifying only the 
 ``--input_model`` parameter in the command line.
 
-.. warning::
+.. note::
 
-   Note that OpenVINO support for Apache MXNet, Caffe, and Kaldi is currently being deprecated and will be removed entirely in the future.
+   Apache MXNet, Caffe, and Kaldi are no longer directly supported by OpenVINO. To make the transition
+   to other frameworks easier, you may keep using them as before, until they are fully discontinued.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_Tutorials.md:49]: I'm not sure I understand how the usage makes the transition easier

File: docs/Documentation/openvino_ecosystem.md [hunk 3]
```diff
@@ -62,7 +62,6 @@ More resources:
 
 **Compile Tool**
```
Comment (tsavina) [inline docs/Documentation/openvino_ecosystem.md:63]: do you want to keep compile tool and dlwb in ecosystem?

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_Tutorials.md (hunk 2 above)
Comment (kblaszczak-intel) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_Tutorials.md:49]: I'm not sure. OVTF could also be moved.

File: docs/Documentation/openvino_ecosystem.md (hunk 3 above)
Comment (kblaszczak-intel) [reply docs/Documentation/openvino_ecosystem.md:63]: Not sure. Same goes for OVTF.

Comment (ilya-lavrenov) [reply docs/Documentation/openvino_ecosystem.md:63]: > do you want to keep compile tool and dlwb in ecosystem?

No

Comment (kblaszczak-intel) [reply docs/Documentation/openvino_ecosystem.md:63]: Ok, I'm easily convinced. Let's throw them out.

File: docs/Documentation/openvino_legacy_features.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    advanced operations on models, such as Model conversion API, Benchmark Tool, 
+    Accuracy Checker, Annotation Converter, Post-Training Optimization Tool, 
+    and Open Model Zoo tools. Most of these tools have been either removed, 
+    replaced by other solutions, or moved to the OpenVINO Runtime package.
+|   :doc:`See how to install Development Tools <openvino_docs_install_guides_install_dev_tools>`
+
+
+| **Model Optimizer**
+|   *New solution:* Direct model support and OpenVINO Converter (OVC)
+|   *Old solution:* Model Optimizer discontinuation planned for 2024
```
Comment (ilya-lavrenov) [inline docs/Documentation/openvino_legacy_features.md:40]: 2025.0

File: docs/Documentation/openvino_legacy_features.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    other model repositories have recently grown in size and popularity,
+    Open Model ZOO will no longer be maintained. You may still use its resources
+    until they are fully removed.  
+|   :doc:`See the Open Model ZOO documentation <model_zoo>`
+|   `Check the OMZ GitHub project <https://github.com/openvinotoolkit/open_model_zoo>`__
+
+
+| **Apache MXNet, Caffe, and Kaldi model formats**
+|   *New solution:* conversion to ONNX will be the only option
+|   *Old solution:* conversion of these models to OpenVINO IR will be unsupported
```
Comment (ilya-lavrenov) [inline docs/Documentation/openvino_legacy_features.md:66]: will be removed in 2024.0

File: docs/Documentation/openvino_legacy_features.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+|   *Old solution:* POT discontinuation planned for 2024
+|    
+|   Neural Network Compression Framework (NNCF) now offers the same functionality as POT,
+    apart from its original feature set. It is currently the default tool for performing 
+    both, post-training and quantization optimizations, while POT is considered deprecated.
+|   :doc:`See the deprecated POT documentation <pot_introduction>` 
+|   :doc:`See how to use NNCF for model optimization <openvino_docs_model_optimization_guide>`
+|   `Check the NNCF GitHub project, including documentation <https://github.com/openvinotoolkit/nncf>`__
+
+| **old API 1.0**
```
Comment (ilya-lavrenov) [inline docs/Documentation/openvino_legacy_features.md:86]:
```suggestion
| **old Inference API 1.0**
```

File: docs/install_guides/installing-openvino-overview.md [hunk 7]
```diff
@@ -52,12 +52,8 @@
       ===============  ==========  ======  =========  ========  ============ ==========
        CPU              V           V        V         V          V           V
        GPU              V           V        V         V          V           V
-       GNA              V           V        V         V          V           V
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-overview.md:56]: let's use vcpkg with small letters above

**Review by as-suvorov — APPROVED** [review]

**Review by tsavina — APPROVED** [review]

File: docs/install_guides/installing-openvino-yum.md [hunk 8]
```diff
@@ -129,7 +129,9 @@ Run the following command:
 
    yum list installed 'openvino*'
 
+.. note::
 
+   You can additionally install Python APIs using one of the alternative methods (:doc:`conda <openvino_docs_install_guides_installing_openvino_conda>` or :doc:`pip <openvino_docs_install_guides_installing_openvino_pip>`).
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-yum.md:134]:
```suggestion
   You can additionally install Python API using one of the alternative methods (:doc:`conda <openvino_docs_install_guides_installing_openvino_conda>` or :doc:`pip <openvino_docs_install_guides_installing_openvino_pip>`).
```

File: docs/install_guides/installing-openvino-vcpkg.md [hunk 9]
```diff
@@ -1,19 +1,18 @@
-# Install OpenVINO™ Runtime via VCPKG {#openvino_docs_install_guides_installing_openvino_vcpkg}
+# Install OpenVINO™ Runtime via vcpkg {#openvino_docs_install_guides_installing_openvino_vcpkg}
 
 @sphinxdirective
 
 .. meta::
    :description: Learn how to install OpenVINO™ Runtime on Windows, Linux, and macOS 
-                 operating systems, using VCPKG.
+                 operating systems, using vcpkg .
```
Comment (tsavina) [inline docs/install_guides/installing-openvino-vcpkg.md:7]:
```suggestion
                 operating systems, using vcpkg.
```

**Pull request merged**